app.rb

```diff
@@ -92,7 +92,7 @@ def book_list
 
 def people_list
   puts 'Choose a person from the list by number:-'
-  i = 0
+  j = 0
```

Comment:
- [ ] Kindly change this line from `j = 0` to `i = 0` to make the app work as it rising errors because of this.

Reply:
oh 
thats funny 
i did this in the bug challange next week and i forget to change it 
i will change it now thanks 
(💯